src/_utils/__tests__/DateUtils-test.js

```diff
@@ -18,3 +18,57 @@ describe('epochToDateString', () => {
         expect(str).toEqual('2001-09-09');
     });
 });
+describe('secondsToTimeString', () => {
+    it('convert seconds to minutes',() =>{
+        const seconds = 60 ;
+        expect(secondsToTimeString(seconds)).toEqual('1 minute(s)');
+    });
+    it('convert seconds to Days,hour, minute and second', ()=>{
+        expect(secondsToTimeString(90177)).toEqual('1 day(s) 1 hour(s)2 minute(s)57 second(s)');
+    });
+
+});
+
+describe('getLastXMonthEpoch', () => {
+    it('Set the x months from the current date and convert time in epoch', ()=>{
+        const currTimeEpoch = (new Date()).getTime()/1000;
+        expect(getLastXMonthEpoch(0)).toEqual(Math.floor(currTimeEpoch));
+    });
+});
+
+describe('epochToDate', () =>{
+    it('convert epoch format to date', () =>{
+        const epochTime = (new Date())/1000;
+        expect(epochToDate(new Date()/1000)).toEqual(new Date());
+    });
+});
+
+describe('dateToEpoch', () => {
+    it('convert date to epoch equivalent', ()=>{
+        expect(dateToEpoch(new Date())).toEqual(Math.floor(new Date()/1000));
+    });
+});
+
+describe('timeStringSmaller', () => {
+    it('Comapare two time strings and determine if the first is smaller', ()=>{
+        expect(timeStringSmaller('010203','020203')).toEqual(true);
```

Comment:
I think your input format is wrong ~ it should be `HH:mm:ss`
Maybe we should do some checking with the input format though, can you fix it?